### DISCUSSION TITLE
CREO on Mac

### DISCUSSION THREAD

**Discussion by nagarjna** [WhatTheBlock/WindowsSimplify#105] [Q&A]
Is it possible to use CREO in an 8gb RAM m1pro

**Comment by WhatTheBlock** [comment]
1. Choose a virtual machine. Such as `VMware Fusion`, `Parallels Desktop`, `VirtualBox`, etc.
2. Install the Windows ARM64 system into the virtual machine. If you don’t know how to operate, you can refer to other tutorials.
3. Install CREO using the Windows system in the virtual machine.

**Reply by nagarjna** [reply]
I used in all three but CREO cannot be opened(although I installed till the very end). The problem I encounter is using windows and Mac simultaneously exhausts my ram upto 80% . Lite windows are not in the format of ARM
(also tried Parallels for fee trial).
Please do help me in finding a lite software that supports CREO in Mac. Also Thanks for the reply🙏🏻
